vispy/visuals/visual.py

```diff
@@ -30,6 +30,11 @@ class Visual(object):
     bounds_change : Event
         Emitted when the bounds of the visual have changed.
 
+    Notes
+    -----
+    -   When used in the scenegraph, all Visual classes are mixed with
```

Comment:
I think just one space after the `-` is standard for restructuredtext bullet lists